code/controllers/master.dm

```diff
@@ -66,7 +66,11 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 			Recover()
 			qdel(Master)
 		else
-			init_subtypes(/datum/controller/subsystem, subsystems)
+			var/list/subsytem_types = subtypesof(/datum/controller/subsystem)
+			sortTim(subsytem_types, /proc/cmp_subsystem_init)
+			var/list/_sss = subsystems
```

Comment:
It's used 2 lines below, it's a performance thing to bring the subsystems list into the stack instead of looking it up on the datum every time